ls1intum/Artemis
# `Development`: Improve documentation about manual assessment

### Checklist
#### General

### Motivation and Context
This PR adds more to the manual assessment documentation.

### Description
Manual assessment documentation now contains the following headlines:
- Assessment Dashboard
- Structured Grading Criteria
- Integrated Training Process
- Double-blind Grading
- Grading Leaderboard

Also, its page is on the top level in the documentation (not hidden below the exercises section).

### Steps for Testing
1. Go to https://artemis-platform--6648.org.readthedocs.build/en/6648/user/assessment.html
2. Carefully read the documentation and provide feedback wherever necessary
3. Make sure that everything is well documented and easy to follow


#### Manual Tests
